src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,23 @@
-export const TodoFilter = () => (
+import React from 'react';
+
+interface Props {
+  setFilter: (value: string) => void;
```

Comment:
if we pass a function to a component that will be called on some event, then the best name for it is to start with a prefix `on`
like this `onSetFilter` or `onChangeFilter`, `onChangeSearchValue`, etc